modules/exploits/windows/http/ca_arcserve_rpc_authbypass.rb

```diff
@@ -105,16 +131,13 @@ def exploit
     end
 
     # report the auth
-    auth = {
-        :host   => datastore['RHOST'],
-        :port   => 445,
-        :sname  => 'smb',
-        :proto  => 'tcp',
-        :user   => user,
-        :pass   => pass,
-        :active => true
-      }
-    report_auth_info(auth)
+    report_cred(
```

Comment:
I feel like the original `report_auth_info` was located at the bad place.

After report_cred you can read:

```ruby
    if user.nil? or pass.nil?
      print_error("Failed to collect the username and password")
      return
    end

```

Probably doesn't worth to report_cred until this check is passed.

On the other hand, the credentials are tried later with `psexec`. So maybe you'd like to wait for reporting until psexec is tried. If psexec works, you can report the credentials as `SUCCESSFUL`.